@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1665,22 +1678,27 @@ export class TextElementsRenderer {
                 iconIndex === undefined
             );
             if (placeResult === PlacementResult.Invisible) {
-                if (placementStats) {
+                if (placementStats !== undefined) {
```

Comment:
It's fine as it was. Our coding style guidelines favor implicit conversion to boolean for object types.